rules_poetry/poetry.bzl

```diff
@@ -3,6 +3,13 @@ load(":json_parser.bzl", "json_parse")
 def _clean_name(name):
     return name.lower().replace("-", "_").replace(".", "_")
 
+# Poetry doesn't add several packages in the poetry.lock file:
+# https://github.com/python-poetry/poetry/blob/d2fd581c9a856a5c4e60a25acb95d06d2a963cf2/poetry/puzzle/provider.py#L55
+# See also https://github.com/python-poetry/poetry/issues/1584 for explanations
+def _ignore_poetry_unsafe_pacakges(packages):
+    unsafe_packages = ["setuptools", "distribute", "pip", "wheel"]
+    return [ p for p in packages if p not in unsafe_packages]
```

Comment:
formatting looks a little inconsistent, can you run buildifier and see if it changes anything?